### PR TITLE
Us46 request and issue authentication code via sms

### PR DESCRIPTION
**Pull request by Lightman32**

### CODE REVIEW

File: application/deed/views.py [hunk 1]
```diff
@@ -129,3 +132,50 @@ def sign_deed(deed_reference):
...
+    if borrower_token is not None and borrower_token != '':
+        borrower = Borrower.get_by_token(borrower_token.strip())
+
+        if borrower is not None:
+            borrower_phone_number = borrower.phonenumber
+
+            code = generate_sms_code(deed_reference, borrower_token)
+
+            message = code + " is your digital mortgage authentication code."
+            twilio_phone_number = config.TWILIO_PHONE_NUMBER
```
Comment (Kirosoft) [inline application/deed/views.py]: my preference would be to assign the constant directly into the literal below, as the variable is otherwise not used.

Comment (Lightman32) [inline application/deed/views.py]: I agree, will be changed in next commit.

File: application/deed/views.py [hunk 2]
```diff
@@ -129,3 +132,50 @@ def sign_deed(deed_reference):
...
+            try:
+                client = TwilioRestClient(config.ACCOUNT_SID, config.AUTH_TOKEN)
+
+                client.messages.create(
+                    to=borrower_phone_number,
+                    from_=twilio_phone_number,
+                    body=message,
+                )
+                LOGGER.error("SMS has been sent to  " + borrower_phone_number)
```
Comment (tsangsimon) [inline application/deed/views.py]: Logger.info for this one

**Comment by tsangsimon** [comment]
:+1:

**Pull request merged**